.circleci/config.yml

```diff
@@ -81,9 +81,6 @@ jobs:
     steps:
       - git-shallow-clone/checkout_advanced:
           clone_options: '--depth 1 --verbose'
-      - run:
-          name: Remove examples
-          command: rm -rf examples/
```

Comment:
😍 This has been the bane of me when debugging differences between local and CI. Thank you for removing!